app/controllers/volunteer_ops_controller.rb

```diff
@@ -5,7 +7,49 @@ class VolunteerOpsController < ApplicationController
   def index
     @volunteer_ops = VolunteerOp.order_by_most_recent
     @organisations = Organisation.where(id: @volunteer_ops.select(:organisation_id))
-    @markers = build_map_markers(@organisations)
+    harrow_markers = build_map_markers(@organisations, :harrow)
+    # Do-it API works from below
+    host = "https://api.do-it.org"
+    href = "/v1/opportunities\?lat\=51.5978\&lng\=-0.3370\&miles\=1 "
+    @doit_orgs = Array.new
+    collect_all_items(host, href, @doit_orgs)
+    doit_markers = build_map_markers(@doit_orgs, :doit)
+    @markers = harrow_markers[0...-1]+', ' + doit_markers[1..-1]
+  end
+ 
+  def build_map_markers(organisations, type)
```

Comment:
No, I do not think so
`build_map_markers` was there before us)
will move them to private